x/stake/client/rest/utils.go

```diff
@@ -24,110 +19,6 @@ func contains(stringSlice []string, txType string) bool {
 	return false
 }
 
-func getDelegatorValidator(cliCtx context.CLIContext, cdc *wire.Codec, delAddr sdk.AccAddress, valAddr sdk.ValAddress) (
```

Comment:
I basically strongly disagree with a lot of bloat that has been added to the staking keeper, it looks like it's maybe a symptom of not having good utils (I see you've removed a lot of them from here) - I understand that after switching to the querier - these utils should maybe be getting used in both the CLI and REST queries - so let's figure out a good place for some of this stuff (which is not in the keeper though!) - OR maybe it could be in the keeper, but in a new `query_utils.go` file